src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -47,12 +52,89 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
     reset({ reviewNotes: '' });
   };
 
+  const overflow: object = {
+    'overflow-y': 'scroll',
+    'max-height': '300px'
+  };
+
+  const proposedSolutionStyle = {
+    cursor: 'pointer',
+    width: 'auto',
+    margin: 'auto',
+    display: 'block'
+  };
+
+  if (cr.type === 'ISSUE') {
```

Comment:
If I understand this correctly, if it's an ISSUE change request, it returns this new modal you've put together, and otherwise it returns the old one. If that's the case then do this with a ternary operator. return (cr.type === 'ISSUE') ? component : component

Reply:
3 types actually have proposed solutions so used an OR instead of ternary.